libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -241,6 +241,31 @@ void AP_Camera_Backend::send_camera_settings(mavlink_channel_t chan) const
         NaN);               // focusLevel float, percentage from 0 to 100, NaN if unknown
 }
 
+// send camera capture status message to GCS
+void AP_Camera_Backend::send_camera_capture_status(mavlink_channel_t chan) const
+{
+    const float NaN = nanf("0x4152");
+
+    const uint32_t time_boot_ms = AP_HAL::millis();
+    const float image_interval = 0.0;     // Image capture interval (s)
+    const uint32_t recording_time_ms = 0; // Time since recording started (ms)
+    const float available_capacity = NaN; // Available storage capacity (MiB)
+    const uint8_t image_status = 0;       // Current status of image capturing (0: idle, 1: capture in progress, 2: interval set but idle, 3: interval set and capture in progress)
+    const uint8_t video_status = 0;       // Current status of video capturing (0: idle, 1: capture in progress)
+    const int32_t image_count = -1;       // Total number of images captured
```

Comment:
image count could be set to "image_index".

Reply:
Not according to this: https://github.com/ArduPilot/mavlink/blob/master/message_definitions/v1.0/common.xml#L5820

Reply:
Oops, I follow now. I though you wanted to rename the variable. But you want to use the existing `image_index` field. Updated.